ql/src/java/org/apache/hadoop/hive/ql/plan/ImportTableDesc.java

```diff
@@ -365,4 +366,8 @@ public void setReplWriteId(Long replWriteId) {
       this.createTblDesc.setReplWriteId(replWriteId);
     }
   }
+
+  public Path getTableLocationPath() {
```

Comment:
This method shouldn't be allowed for virtual view as it doesn't have any location. Shall check it here.

Reply:
ok.